homeassistant/components/switch/template.py

```diff
@@ -29,7 +29,7 @@
 OFF_ACTION = 'turn_off'
 
 SWITCH_SCHEMA = vol.Schema({
-    vol.Required(CONF_VALUE_TEMPLATE): cv.template,
+    vol.Optional(CONF_VALUE_TEMPLATE): cv.template,
```

Comment:
That is not allow. Please revert this change

Reply:
Sorry, I have another pull request open. I managed to get the branches mixed up.